persistence/upgrades/sql-8to9.md

```diff
@@ -0,0 +1,66 @@
+---
+title: SQL Persistence Upgrade Version 8 to 9
+summary: Migration instructions on how to upgrade to SQL Persistence version 9
+reviewed: 2025-12-09
+component: SqlPersistence
+related:
+- persistence/sql
+- nservicebus/upgrades/9to10
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 9
+ - 10
+---
+
+## Script generation via source generators
```

Comment:
I find this slightly too detailed, but on the other hand it is also great background information for any technical reader.